<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsFileDashboard</class>
 <widget class="QWidget" name="ResultsFileDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results File Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="metricsContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="metricsLayout">
      <item>
       <widget class="QGroupBox" name="metric1_group">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="metric1_layout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="metric1_title">
           <property name="text">
            <string>Total Primary Energy (2050)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="metric1_value">
           <property name="text">
            <string>10.5 TOE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>color: #2E8B57;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="metric2_group">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="metric2_layout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="metric2_title">
           <property name="text">
            <string>Total Electricity (2050)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="metric2_value">
           <property name="text">
            <string>72.4 TWh</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>color: #2E8B57;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="metric3_group">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="metric3_layout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="metric3_title">
           <property name="text">
            <string>% Clean Electricity (2050)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="metric3_value">
           <property name="text">
            <string>72.3%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>color: #2E8B57;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="metric4_group">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="metric4_layout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="metric4_title">
           <property name="text">
            <string>Total Emissions (2050)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="metric4_value">
           <property name="text">
            <string>10.2 ktCO2e</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>color: #2E8B57;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="dashboardTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="overviewTab">
      <attribute name="title">
       <string>Overview</string>
      </attribute>
      <layout class="QVBoxLayout" name="overviewLayout">
       <item>
        <layout class="QGridLayout" name="overviewChartsGrid">
         <item row="0" column="0">
          <widget class="QWidget" name="chart1_widget" native="true">
           <layout class="QVBoxLayout" name="chart1_layout">
            <item>
             <widget class="QLabel" name="chart1_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Primary Energy Supply (PJ)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWebEngineView" name="primary_energy_demand_chart">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="chart2_widget" native="true">
           <layout class="QVBoxLayout" name="chart2_layout">
            <item>
             <widget class="QLabel" name="chart2_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Electricity Generation (TWh)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWebEngineView" name="electricity_generation_chart">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="chart3_widget" native="true">
           <layout class="QVBoxLayout" name="chart3_layout">
            <item>
             <widget class="QLabel" name="chart3_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Primary Energy Mix (2050)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWebEngineView" name="primary_energy_pie_chart">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="chart4_widget" native="true">
           <layout class="QVBoxLayout" name="chart4_layout">
            <item>
             <widget class="QLabel" name="chart4_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Electricity Sources (2050)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWebEngineView" name="electricity_pie_chart">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="electricityTab">
      <attribute name="title">
       <string>Electricity</string>
      </attribute>
      <layout class="QVBoxLayout" name="electricityLayout">
       <item>
        <layout class="QHBoxLayout" name="electricityChartsLayout">
         <item>
          <widget class="QWidget" name="electricity_chart1_widget" native="true">
           <layout class="QVBoxLayout" name="electricity_chart1_layout">
            <item>
             <widget class="QLabel" name="electricity_chart1_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Electricity Generation by Fuel Source</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWebEngineView" name="electricity_generation_by_fuel_chart">
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>250</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="electricity_chart2_widget" native="true">
           <layout class="QVBoxLayout" name="electricity_chart2_layout">
            <item>
             <widget class="QLabel" name="electricity_chart2_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Electricity Costs by Fuel Source</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWebEngineView" name="electricity_costs_by_fuel_chart">
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>250</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="cost_breakdown_group">
         <property name="title">
          <string>Electricity Cost Breakdown ($/MWh)</string>
         </property>
         <layout class="QVBoxLayout" name="cost_breakdown_layout">
          <item>
           <widget class="QTableWidget" name="cost_breakdown_table">
            <property name="minimumSize">
             <size>
              <width>700</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebEngineWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
